free-programming-books-es.md

```diff
@@ -330,11 +340,10 @@
 
 ### SQL
 
-* [El lenguaje SQL](http://ocw.uoc.edu/computer-science-technology-and-multimedia/bases-de-datos/bases-de-datos/P06_M2109_02149.pdf) (PDF)
-* [Manual de gestión de base de datos](http://www.jorgesanchez.net/bd/gbd2012.pdf) (PDF)
-* [Tutorial de SQL](http://www.desarrolloweb.com/manuales/9/)
+* [Manual de SQL](http://jorgesanchez.net/manuales/sql/intro-sql-sql2016.html) (HTML)
+* [Tutorial de SQL](http://www.desarrolloweb.com/manuales/9/) (HTML)
 
 
 ### SVG
 
-* [SCALABLE](https://leanpub.com/scalable/) (PDF) (EPUB) (MOBI)
+* [Scalable. Un libro sobre SVG](https://leanpub.com/scalable/) (LeanPub)
```

Comment:
should probably change to `*(Leanpub account required)*` but translated into spanish